2019/questions.md

```diff
@@ -386,9 +386,31 @@
 * Other
 >LH Note: who is supposed to get this question?
 
-41. Thank you for taking the time to answer our questions. If you would like a copy of the results, please give us your email address. We will only use this email address to communicate with you about this survey.
+41. To what degree do you perceive the following companies to be “good open source community citizens” in terms of their contributions, collaboration and leadership on open source projects and initiatives within the CNCF ecosystem? 
+>{Excellent Above Average Average Below Average Very Poor Don't know}
```

Comment:
Take Comcast out and add Huawei?  Note that I took "Red Hat"" out because it merged with IBM. We can add that back in if you want, but I don't think there will be a lot of good comparisons between Red Hat vs IBM unless you ask specifically about that.

Reply:
Sort alphabetically

Reply:
what does CNCF have to do with this? please just make this about open source